android/BOINC/app/src/main/res/values/strings.xml

```diff
@@ -65,7 +65,7 @@
     <string name="attachproject_login_header_description">Description:</string>
     <string name="attachproject_login_header_home">Home:</string>
     <string name="attachproject_login_header_url">Website:</string>
-    <string name="attachproject_login_header_id_email">eMail:</string>
+    <string name="attachproject_login_header_id_email">Email address:</string>
```

Comment:
I can accept changes to this file (and to the file from the `values-en` folder), but all other files come from the translation service: https://app.transifex.com/boinc/
Please remove changes from all files except two I mentioned above

Reply:
Done!